packages/addresses/src/program-derived-address.ts

```diff
@@ -26,7 +80,10 @@ const PDA_MARKER_BYTES = [
 // TODO: Coded error.
 class PointOnCurveError extends Error {}
 
-async function createProgramDerivedAddress({ programAddress, seeds }: PDAInput): Promise<Base58EncodedAddress> {
+async function createProgramDerivedAddress({
+    programAddress,
```

Comment:
Just a little side note: Seems we want to move away from `programId` and instead favor `programAddress`? There might be some stuff in `rpc-core` to update then

Reply:
That's a good point.